controllers/reaction_controller.py

```diff
@@ -1,10 +1,17 @@
 from discord import Message
 from db import DB
+from datetime import datetime, timedelta
 
 
 class ReactionController:
     @staticmethod
     async def apply_reactions(message: Message):
         emojis = DB().get_reactions_for_user(message.author.id)
-        for emoji in emojis:
-            await message.add_reaction(emoji)
+        if len(emojis) != 0: # prevents further DB calls if user does not have any Robomojis
+            emoji_delay_seconds = DB().get_emoji_reaction_delay()
+            last_reaction_datetime = DB().get_emoji_reaction_last_used(message.author.id)
+            robomoji_allowed_datetime = last_reaction_datetime + timedelta(seconds=emoji_delay_seconds)
+            if (robomoji_allowed_datetime <= datetime.now()):
```

Comment:
super nit: This should get taken care of on merge with our linter, but for future reference no need for parentheses here